app/templates/_package.json

```diff
@@ -84,7 +84,6 @@
   },
   "devDependencies": {
     "aliv": "^1.6.0",
-    "karma-ng-html2js-preprocessor": "^0.1.2",
```

Comment:
That's needed for tests directives/components that use templateUrl instead of template:

https://github.com/ericmdantas/generator-ng-fullstack/blob/master/app/templates/_karma.conf.js#L53